src/ServiceControl/Infrastructure/NServiceBusFactory.cs

```diff
@@ -44,6 +44,7 @@ public static IStartableBus Create(Settings.Settings settings, IContainer contai
 
             configuration.Conventions().DefiningEventsAs(t => typeof(IEvent).IsAssignableFrom(t) || IsExternalContract(t));
             configuration.EndpointName(settings.ServiceName);
+            configuration.ReportCustomChecksTo(settings.ServiceName);
```

Comment:
Question: is this line going to cover ASB custom check plugin as well?

Reply:
Yes, ASB will send the custom check messages to the main ServiceControl queue that is created when you add an ASB instance of ServiceControl